src/scene/scenegraphnode.cpp

```diff
@@ -193,6 +193,15 @@ namespace {
         openspace::properties::Property::Visibility::Hidden
     };
 
+    constexpr openspace::properties::Property::PropertyInfo GuiFocussableInfo = {
+        "IsFocussable",
+        "Focussable Hint",
+        "This value serves as a hint to determine if it makes sense to focus the camera "
```

Comment:
This description should mention that it affects how the object appears in the GUI, if that's what we want to use it for

It would be an easy addition if we want to include it in the new UI